afs/media/js/views/components/reports/scenes/existence.js

```diff
@@ -152,9 +167,11 @@ define(['underscore', 'knockout', 'arches', 'utils/report','bindings/datatable']
                         if(parts?.length){
                             var partObservables = parts.map(x => {
```

Comment:
Also noticed this var - use const or let.

Reply:
Fixed thanks.